content/applications/general/integrations/translate.rst

```diff
@@ -0,0 +1,122 @@
+
+.. tip::
+   If the *Google Developer Console* has existing projects, click the drop-down menu next to the
+   :guilabel:`Google Cloud` icon, and a pop-over window appears. Next, click :guilabel:`New Project`
+   top-right of the pop-over window.
+
+On the :menuselection:`New Project` screen, rename the :guilabel:`Project name` to `Odoo Translate`,
+and browse for the :guilabel:`Location`. Set the :guilabel:`Location` as the *Google Workspace
+organization*. If a personal Gmail account is being used, leave the :guilabel:`Location` as
+:guilabel:`No Organization`.
+
+.. image:: translate/new-project.png
+   :align: center
+   :alt: Project Name and Location for Google OAuth.
+
+Click on :guilabel:`Create` to finish this step.
+
+API Library
+-----------
+
+Next, the :guilabel:`Cloud Translation API` needs to be installed on this newly-created project.
+To do that, click :guilabel:`Library` in the left menu. Then, search the term `Cloud Translation
+API`, and click into the result. This should be a *Google Enterprise API* labeled `Cloud Translation
+API`.
+
+Click :guilabel:`Enable` to install the library on this project.
+
+.. important::
+   Using the *Google Translate* API **requires** a current billing account with `Google
+   <https://https://myaccount.google.com/>`_.
+
+Once a billing account is setup with *Google* and the library is enabled, click :guilabel:`Manage`
+to finish configuration on the API.
+
+Create Credentials
+------------------
+
+Now that the project is set up, and the library is enabled, credentials **must** be created. This
+includes the *API key*.
+
+To begin this process, click :guilabel:`Credentials` in the left sidebar menu.
+
+Then, click :guilabel:`Create Credentials` in the top menu, and select :guilabel:`API key` from the
+drop-down menu.
+
+.. image:: translate/api-key.png
+   :align: center
+   :alt: Create an API key in the Google developer console.
+
+Copy the :guilabel:`API key` for use in the next section.
+
+.. warning::
+   For security purposes, the usage of the *API key* can be restricted.
+
+   To do that, go to the :guilabel:`API restrictions` by clicking on :guilabel:`Edit API key` in the
+   pop-over window, or on the listed :guilabel:`API key` on the :guilabel:`Credentials` page. From
+   here, key restrictions can be set. This includes setting an application to restrict the use of
+   the API key, and whether this *API key* can call any API.
+
+   It is recommended that the Odoo *Translate API* be restricted to **only** allow requests from the
+   configured database and to the `Cloud Translation API`.
+
+   To add the website restriction, click :guilabel:`Websites`, under the :guilabel:`Set an
+   application restriction`. Then, enter the address of the database *Google Translate* is being
+   used in, by clicking on :guilabel:`Add`. Lastly, add the :abbr:`URL (Uniform Resource Locator)`,
+   and click :guilabel:`Done`.
```

Comment:
Are :guilabel:s missing here for Address and URL fields?
```suggestion
   To add the website restriction, click :guilabel:`Websites`, under :guilabel:`Set an application
   restriction`. Then, enter the :guilabel:`Address` of the database *Google Translate* is being
   used in, by clicking on :guilabel:`Add`. Lastly, add the :guilabel:`URL`, and click
   :guilabel:`Done`.
```

Reply:
No they are not missing